root/edit/details/add_isrcs.tt

```diff
@@ -2,7 +2,7 @@
   [% IF edit.data.client_version.defined %]
   <tr>
     <th>[% l('Client:') %]</th>
-    <td>[% (edit.data.client_version || l('(unknown)')) | html %]</td>
+    <td>[% (edit.data.client_version || lp('(unknown)', 'isrc_client')) | html %]</td>
```

Comment:
This is probably trivial, but the underscore in the context seems weird to me. We only have underscores elsewhere where it uses the DB table name as context, but this isn't a table name or field, just a phrase. :)
```suggestion
    <td>[% (edit.data.client_version || lp('(unknown)', 'isrc client')) | html %]</td>
```

Reply:
This applies to other context strings too: `medium_forrmat` :arrow_forward: `medium format`, and so on.

Reply:
Will do! I honestly didn't even know we could have spaces on those :)